static/app/views/releases/list/index.tsx

```diff
@@ -244,9 +297,60 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAlertBanner() {
+    const {selection, organization} = this.props;
+    const {hasSessions} = this.state;
+
+    const selectedProjectId =
+      selection.projects && selection.projects.length === 1 && selection.projects[0];
+    const selectedProject = organization.projects.find(
+      p => p.id === `${selectedProjectId}`
+    );
+
+    if (!selectedProject) {
+      return null;
+    }
+
+    return (
+      <Feature features={['organizations:release-adoption-chart']}>
+        <Projects orgId={organization.slug} slugs={[selectedProject.slug]}>
+          {({projects, initiallyLoaded, fetchError}) => {
+            const project = projects && projects.length === 1 && projects[0];
+            const projectCanHaveReleases =
+              project && project.platform && releaseHealth.includes(project.platform);
+
+            if (
+              !initiallyLoaded ||
+              fetchError ||
+              !projectCanHaveReleases ||
+              hasSessions
+            ) {
+              return null;
+            }
```

Comment:
This needs adjusting because otherwise, the banner will show up for a second when this page is loading and also will be there on unsupported platforms.